src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background: url("images/star.svg") center no-repeat;
+}
+
+.stars__star:first-child {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-last-child(n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-last-child(n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-last-child(n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-last-child(n) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-last-child(n+4),
.stars--3 :nth-last-child(n+3),
.stars--4 :nth-last-child(n+2),
.stars--5 :nth-last-child(n) {
  background-image: url(images/star-active.svg);
}
```
Це все можна об'єднати

---

Я не знаю як саме, думав про об'єднання але не зміг знати рішення.